cloud/amazon/redshift.py

```diff
@@ -0,0 +1,526 @@
+    required: false
+    aliases: ['zone', 'aws_zone']
+  number_of_nodes:
+    description:
+      - Number of nodes. Only used when cluster_type=multi-node.
+    required: false
+    default: null
+  cluster_subnet_group_name:
+    description:
+      - which subnet to place the cluster
+    required: false
+    aliases: ['subnet']
+  cluster_security_groups:
+    description:
+      - in which security group the cluster belongs
+    required: false
+    default: null
+    aliases: ['security_groups']
+  vpc_security_group_ids:
+    description:
+      - VPC security group
+    required: false
+    aliases: ['vpc_security_groups']
+    default: null
+  preferred_maintenance_window:
+    description:
+      - maintenance window
+    required: false
+    aliases: ['maintance_window', 'maint_window']
+    default: null
+  cluster_parameter_group_name:
+    description:
+      - name of the cluster parameter group
+    required: false
+    aliases: ['param_group_name']
+    default: null
+  automated_snapshot_retention_period:
+    description:
+      - period when the snapshot take place
+    required: false
+    aliases: ['retention_period']
+    default: null
+  port:
+    description:
+      - which port the cluster is listining
+    required: false
+    default: null
+  cluster_version:
+    description:
+      - which version the cluster should have
+    required: false
+    aliases: ['version']
+    choices: ['1.0']
+    default: null
+  allow_version_upgrade:
+    description:
+      - flag to determinate if upgrade of version is possible
+    required: false
+    aliases: ['version_upgrade']
+    default: null
+  number_of_nodes:
+    description:
+      - number of the nodes the cluster should run
+    required: false
+    default: null
+  publicly_accessible:
+    description:
+      - if the cluster is accessible publicly or not
+    required: false
+    default: null
+  encrypted:
+    description:
+      -  if the cluster is encrypted or not
+    required: false
+    default: null
+  elastic_ip:
+    description:
+      - if the cluster has an elastic IP or not
+    required: false
+    default: null
+  new_cluster_identifier:
+    description:
+      - Only used when command=modify.
+    required: false
+    aliases: ['new_identifier']
+    default: null
+  wait:
+    description:
+      - When command=create, modify or restore then wait for the database to enter the 'available' state. When command=delete wait for the database to be terminated.
+    required: false
+    default: "no"
+    choices: [ "yes", "no" ]
+  wait_timeout:
+    description:
+      - how long before wait gives up, in seconds
+    default: 300
+requirements: [ 'boto' ]
+extends_documentation_fragment: aws
+'''
+
+EXAMPLES = '''
+# Basic cluster provisioning example
+- redshift: >
+    command=create
+    node_type=dw1.xlarge
+    identifier=new_cluster
+    username=cluster_admin
+    password=1nsecure
+'''
+
+RETURN = '''
+changed:
```

Comment:
I assume changed isn't needed to be documented as it's a standard Ansible return value.

Thanks for adding the rest though